src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -242,8 +242,9 @@ public Result listSubscriptions(@Nullable final String owningApplication, @Nulla
             }
             final String total = topicPartition.getOffset();
             if (!Cursor.BEFORE_OLDEST_OFFSET.equals(total)) {
-                final long clientOffset = zkSubscriptionClient.getOffset(partition.getKey());
-                unconsumedEvents = Long.valueOf(total) - clientOffset;
+                final String clientOffset = zkSubscriptionClient.getOffset(partition.getKey());
+                // TODO! Use different service for this stuff
+                unconsumedEvents = Long.valueOf(total) - Long.valueOf(clientOffset);
```

Comment:
As soon as we merge my PR with cursor comparison, this will be implemented there.